variables.tf

```diff
@@ -306,6 +306,17 @@ check "deprecations" {
   }
 }
 
+check "imds_hop_count" {
+  assert {
+    condition     = !anytrue([for filter in var.aws_ami_filter : can(regex("rubrik-mp-cc-[0-8]\\*", filter))]) || var.metadata_http_put_response_hop_limit != 2
```

Comment:
Discussed on a call, the regex needs to be updated to handle minor CDM versions and version `>9.x.x`. A regex like `"rubrik-mp-cc-[6-8]"` will probably work.